Sample/src/androidTest/java/dev/testify/sample/TestingResourceConfigurationsExampleTest.kt

```diff
@@ -35,7 +35,7 @@ import dev.testify.sample.test.clientDetailsView
 import dev.testify.sample.test.getViewState
 import org.junit.Rule
 import org.junit.Test
-import java.util.Locale
+import java.util.*
```

Comment:
Just wanted to check that this was intentional

Reply:
It's not. Stupid upgrade to Android Studio broke my settings. I'll fix this.